usage/camera/raspicam/README.md

```diff
@@ -1,6 +1,6 @@
-# raspicam commands
+# Raspicam commands
 
-`raspistill`, `raspivid` and `raspiyuv` are command line tools for using the camera module.
+`Raspistill`, `raspivid` and `raspiyuv` are command line tools for using the camera module.
```

Comment:
This is the name of a command, and so `raspistill` should be kept in lower-case.

Reply:
OK, I will revert all of these.